scripts/eslint.py

```diff
@@ -0,0 +1,63 @@
+"""  # pylint: disable=django-not-configured
+Check code quality using pycodestyle, pylint, and diff_quality.
+"""
+
+import re
+import subprocess
+import shlex
+import sys
+
+
+def fail_quality(name, message):
+    """
+    Fail the specified quality check.
+    """
+    print(name)
+    print(message)
+    sys.exit()
+
+
+def run_eslint():
+    """
+    Runs eslint on static asset directories.
+    If limit option is passed, fails build if more violations than the limit are found.
+    """
+    violations_limit = 1213
+
+    command = [
+        "node",
+        "--max_old_space_size=4096",
+        "node_modules/.bin/eslint",
+        "--ext", ".js",
+        "--ext", ".jsx",
+        "--format=compact",
+        "."
```

Comment:
@salman2013 
```suggestion
        "lms",
        "cms",
        "common",
        "openedx",
        "xmodule",     
```
I haven't tested it yet, but perhaps specifying each root path would lower the violation count. Specifying `"."` (current directory) might be including node_modules accidentally.

Reply:
@kdmccormick I believe your thoughts are right, making these changes the error count reduced to 1285 from 1303.

<img width="1083" alt="Screenshot 2024-12-06 at 3 29 49 PM" src="https://github.com/user-attachments/assets/a29c5931-2b1c-4d39-b06d-7bb6f18059bd">